WowPacketParserModule.V5_4_0_17359/Parsers/MovementHandler.cs

```diff
@@ -15,11 +16,12 @@ public static class MovementHandler
         [Parser(Opcode.SMSG_LOGIN_SET_TIME_SPEED)]
         public static void HandleUnknown431(Packet packet)
         {
-            packet.ReadSingle("Game Speed");
-            packet.ReadPackedTime("Game Time");
+            PacketLoginSetTimeSpeed setTime = packet.Holder.LoginSetTimeSpeed = new();
+            setTime.NewSpeed = packet.ReadSingle("Game Speed");
+            setTime.GameTime = packet.ReadPackedTime("Game Time").ToUniversalTime().ToUniversalTime().ToTimestamp();
             packet.ReadInt32("Unk 1");
             packet.ReadInt32("Unk 2");
-            packet.ReadPackedTime("Game Time?");
+            setTime.ServerTime = packet.ReadPackedTime("Game Time?").ToUniversalTime().ToUniversalTime().ToTimestamp();
```

Comment:
are we sure this is ServerTime?

Reply:
we are not sure (at least I am not) 🤷‍♂️ but it doesn't matter, while using the proto let's just check both times I guess